components/snackbar/snackbar.html

```diff
@@ -0,0 +1,110 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
```

Comment:
You can keep this in generic or common css files instead of copying in all files.